autoload/fzf/vim.vim

```diff
@@ -964,7 +964,7 @@ function! fzf#vim#snippets(...)
   let colored = map(aligned, 's:yellow(v:val[0])."\t".v:val[1]')
   return s:fzf('snippets', {
   \ 'source':  colored,
-  \ 'options': '--ansi --tiebreak=index +m -n 1 -d "\t"',
+  \ 'options': '--ansi --tiebreak=index +m -n 1,2 -d "\t"',
```

Comment:
should that be ok ^

Reply:
No, I meant the literal `..` which means the whole range.

* `-n 1,..`: Let fzf search either on the first token only or on the whole string.